pkg/bitwarden/cli.go

```diff
@@ -82,7 +82,7 @@ func (c *cliClient) loginAndListItems() error {
 			}
 			err = json.Unmarshal(out, &items)
 			if err != nil {
-				return err
+				return fmt.Errorf("failed to parse bw item list output %s: %v", out, err)
```

Comment:
my compiler usually throws a warning if I use %v instead of %w for errors. but I guess it doesn't matter since it doesn't look like we use Is or As in our code anyway.

Reply:
Fixed in the other PR